src/app/static/src/js/components/documents/fileIcon.vue

```diff
@@ -7,6 +7,6 @@
 
 <script>
     export default {
-        name: "fileIcon"
```

Comment:
Unrelated to changeset, guess this is cos I ran `npm run lint -- --fix` and it fixed this warning.